doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -974,32 +974,42 @@ Generation of induction principles with ``Scheme``
    .. insertprodn scheme_kind sort_family
 
    .. prodn::
-      scheme_kind ::= Equality for @reference
-      | {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
-      sort_family ::= Set
-      | Prop
+      scheme_kind ::= {| Induction | Minimality | Elimination | Case } for @reference Sort @sort_family
+      sort_family ::= Prop
       | SProp
+      | Set
       | Type
 
-  A high-level tool for automatically generating
-  (possibly mutual) :term:`induction principles <induction principle>`
-  for given types and sorts.
-  Each :n:`@reference` is a different inductive type identifier belonging to
-  the same package of mutual inductive definitions.
-  The command generates the :n:`@ident`\s as mutually recursive
-  definitions. Each term :n:`@ident` proves a general principle of mutual
-  induction for objects in type :n:`@reference`.
+   Generates an :term:`induction principles <induction principle>` with given
+   :n:`scheme_type`\s and :n:`scheme_sort`\s for an inductive type. In the case
+   where the inductive definition is a mutual inductive definition, the
+   :n:`with` clause is used to generate a mutually recursive inductive scheme
+   for each clause of the mutual inductive type.
 
-  :n:`@ident`
-    The name of the scheme. If not provided, the scheme name will be determined automatically
-    from the sorts involved.
+   :n:`@ident`
+      The name of the scheme. If not provided, the name will be determined
+      automatically from the :n:`@scheme_type` and :n:`@sort_family`.
```

Comment:
```suggestion
      automatically from the scheme type and :n:`@sort_family`.
```

Fails because `scheme_type` is no longer a nonterminal in the grammar.